TVNZ OnDemand/dist/metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+    "author": {
+      "name": "Cairo",
+      "id": "208105877838888960"
+    },
+    "url": [
+      "www.tvnz.co.nz"
+    ],
```

Comment:
url shouldnt be in an array if it is only 1

Reply:
> url shouldnt be in an array if it is only 1

Thanks for pointing that out, it originally contained a non-www url as well but I removed it and forgot to remove the array.